internal/callgraph/language/golang/callgraph.go

```diff
@@ -0,0 +1,328 @@
+package golang
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/debricked/cli/internal/callgraph/cgexec"
+	"github.com/debricked/cli/internal/callgraph/model"
+	ioFs "github.com/debricked/cli/internal/io"
+	"golang.org/x/tools/go/callgraph"
+	"golang.org/x/tools/go/callgraph/cha"
+	"golang.org/x/tools/go/callgraph/rta"
+	"golang.org/x/tools/go/callgraph/static"
+	"golang.org/x/tools/go/callgraph/vta"
+	"golang.org/x/tools/go/packages"
+	"golang.org/x/tools/go/ssa"
+	"golang.org/x/tools/go/ssa/ssautil"
+)
+
+type ICallgraph interface {
+	RunCallGraph() (string, error)
+}
+
+type Callgraph struct {
```

Comment:
I think CallGraph may be the wrong name here, is this struct a callgraph? Maybe something more explicit like CallGraphGenerator that explains the purpose of the struct and functions would be better?

Reply:
That sounds good!